rules/generate_rules.py

```diff
@@ -0,0 +1,71 @@
+import json
+import re
+import sys
+from pathlib import Path
+
+from jinja2 import Environment
+
+rules_src = Path(__file__).parent / "rules.json"
+if not rules_src.exists():
```

Comment:
What's the use case ? Why not failing?

Reply:
It's needed for the "first pass" of Docker build, where rules are not yet available / we do not want to generate this because we do not have yet the proper folder structure. Pretty harmless anyway